specs/waku/waku.md

```diff
@@ -222,9 +221,16 @@ The status message MUST contain an association list containing various options.
 
 This packet is used for sending the standard Waku envelopes.
 
-#### PoW Requirement
+#### Status Update
+
+The Status Update message is used to communicate an update of the settings of the node.
+The format is the same as the Status message, all fields are optional.
+Fields that are omitted are considered unchanged, fields that haven't changed SHOULD not 
+be transmitted.
 
-This packet is used by Waku nodes for dynamic adjustment of their individual PoW requirements. Recipient of this message should no longer deliver the sender messages with PoW lower than specified in this message.
+**PoW Requirement update**
+
+When PoW is updated, peers SHOULD no longer deliver the sender envelopes with PoW lower than specified in this message.
```

Comment:
Or `MUST NOT`? Currently if that happens, a peer violating this rule is disconnected immediately, at least in Go implementation.